docs/release-notes/next-fortuna/PT-162539245-monitoring

```diff
@@ -1 +1 @@
-* Add `aemon` application to handle network monitoring.
+* Adds network monitoring capability to the node. This is disabled by default. Please refer to the docs/monitoring.md document for the intended usage.
```

Comment:
Is this planned to be present in release notes of 2.5.0 or 3.0.0? I assume 2.5.0. So location of file shall be `docs/release-notes/next/` - not `next-fortuna`.

Reply:
correct, i’ll move it 👌